unit/Makefile

```diff
@@ -16,6 +16,11 @@ SRC += analyses/ai/ai.cpp \
        analyses/does_remove_const/is_type_at_least_as_const_as.cpp \
        compound_block_locations.cpp \
        goto-programs/goto_trace_output.cpp \
+       goto-programs/goto_program_assume.cpp \
+       goto-programs/goto_program_dead.cpp \
+       goto-programs/goto_program_declaration.cpp \
+       goto-programs/goto_program_function_call.cpp \
+       goto-programs/goto_program_goto_target.cpp \
```

Comment:
Lexicographic ordering, please. (goto_program*<goto_trace_output)

Reply:
@tautschnig I fixed the ordering. As to the above `evaluates_to_boolean` function: that PR hasn't been made yet. I removed it from here, and will put it to a PR once we actually make it.